mkdocs/themes/mkdocs/css/base.css

```diff
@@ -117,13 +117,21 @@ a:hover code, a:focus code {
     color: #157AB5;
 }
 
+centered {
```

Comment:
not a really big deal - but a class name of `text-center` would be more semantically correct.